webui-src/app/scss/components/_navbar.scss

```diff
@@ -1,37 +1,82 @@
+@use '../abstracts/' as *;
+
 /* Navbar */
-.nav-logo {
-  width: 130px;
-  padding: 1.2rem 0;
-  display: flex;
-  gap: 0.3rem;
-  align-items: center;
-}
-.nav-logo img {
-  width: 1.6rem;
-}
-.nav-logo h4 {
-  color: white;
-  align-self: end;
-}
-nav.tab-menu {
-  border-right: 4px solid #3ba4d7;
-  background-color: #14141B;
-  box-shadow: 0 5px 5px #aaa;
+.menu {
```

Comment:
```suggestion
nav.menu {
```

Reply:
> 

I have used `&` selector inside it and it breaks when doing this. So, I instead renamed `menu` to `.nav-menu`. Will it be good?

Reply:
Okay, sounds good.